<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="start_game">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>330</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Start new game</string>
    </property>
   </widget>
   <widget class="QLabel" name="info">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Game</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="select">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>350</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>HUMAN VS AI</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AI VS AI</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="b_0">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>220</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(155, 207, 255);
BORDER:2PX SOLID BLACK;
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="b_1">
    <property name="geometry">
     <rect>
      <x>248</x>
      <y>220</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(155, 207, 255);
BORDER:2PX SOLID BLACK;
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="b_2">
    <property name="geometry">
     <rect>
      <x>334</x>
      <y>220</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(155, 207, 255);
BORDER:2PX SOLID BLACK;
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="b_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>220</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(155, 207, 255);
BORDER:2PX SOLID BLACK;
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="b_4">
    <property name="geometry">
     <rect>
      <x>506</x>
      <y>220</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(155, 207, 255);
BORDER:2PX SOLID BLACK;
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="b_5">
    <property name="geometry">
     <rect>
      <x>592</x>
      <y>220</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(155, 207, 255);
BORDER:2PX SOLID BLACK;
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="b_studnia">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>160</y>
      <width>80</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 127);
BORDER:2PX SOLID BLACK;
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="a_studnia">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>80</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 127);
BORDER:2PX SOLID BLACK;
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="a_3">
    <property name="geometry">
     <rect>
      <x>334</x>
      <y>132</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(155, 207, 255);
BORDER:2PX SOLID BLACK;
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="a_5">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>132</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(155, 207, 255);
BORDER:2PX SOLID BLACK;
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="a_1">
    <property name="geometry">
     <rect>
      <x>506</x>
      <y>132</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(155, 207, 255);
BORDER:2PX SOLID BLACK;
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="a_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>132</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(155, 207, 255);
BORDER:2PX SOLID BLACK;
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="a_4">
    <property name="geometry">
     <rect>
      <x>248</x>
      <y>132</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(155, 207, 255);
BORDER:2PX SOLID BLACK;
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="a_0">
    <property name="geometry">
     <rect>
      <x>592</x>
      <y>132</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(155, 207, 255);
BORDER:2PX SOLID BLACK;
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>816</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
